.claude/TM_COMMANDS_GUIDE.md

```
@@ -0,0 +1,147 @@
+## Common Workflows
+
+### Daily Development
+```
+/project:tm/workflows/smart-flow morning
+/project:tm/next
+/project:tm/set-status/to-in-progress <id>
+/project:tm/set-status/to-done <id>
+```
+
+### Task Breakdown
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)